packages/web/src/pages/settings-page/components/desktop/DeveloperApps/DeveloperAppListItem.tsx

```diff
@@ -56,25 +57,33 @@ export const DeveloperAppListItem = (props: DeveloperAppListItemProps) => {
 
   return (
     <li className={styles.listItem}>
-      <span className={styles.listItemIndex}>{index}</span>
+      <span className={styles.listItemImage}>
+        {imageUrl ? (
+          <img width='56' src={imageUrl} />
+        ) : (
+          <IconEmbed color='subdued' size='2xl' />
+        )}
+      </span>
       {divider}
       <span className={styles.listItemAppName}>{name}</span>
       {divider}
-      <PopupMenu
-        items={menuItems}
-        renderTrigger={(ref, onClick, triggerProps) => (
-          <IconButton
-            // @ts-ignore
-            ref={ref}
-            {...triggerProps}
-            aria-label={messages.appActionsLabel}
-            color='default'
-            size='m'
-            icon={IconKebabHorizontal}
-            onClick={() => onClick()}
-          />
-        )}
-      />
+      <span className={styles.listItemOptions}>
+        <PopupMenu
+          items={menuItems}
+          renderTrigger={(ref, onClick, triggerProps) => (
+            <IconButton
+              // @ts-ignore
+              ref={ref}
+              {...triggerProps}
+              aria-label={messages.appActionsLabel}
+              color='subdued'
+              size='m'
+              icon={IconKebabHorizontal}
+              onClick={() => onClick()}
```

Comment:
I know you didn't write the original code, but confused why we create an arrow function here just to call another function instead of passing it in directly...

Reply:
some lazy type stuff. don't want to overscope this change, but yeah it could be fixed w/ proper args on the callbacks